### DISCUSSION TITLE
Enquiry of Offline Hubs Room

### DISCUSSION THREAD

**Discussion by qiangjiadong** [mozilla/hubs#3719] [General Questions & Discussion] [answered]
Hi Guys, I've been planning to present 3 hubs room (all connected via link function through Spoke) for an exhibition.

The internet and location of exhibition is not ideal to present my Hubs rooms constantly online and stable.

May I ask if there is a way to allow the visitors to explore Hubs room and link to 2 other room COMPLETELY through offline mode?

or any other suggestions?

Thks a lot!!!

**Comment by robinkwilson** [comment, marked as answer]
Hello! Unfortunately if the instance is in offline mode, no hubs rooms will be accessible. 

Does that answer your question? If it does not, please give me a little more information on what you're looking to achieve. I might not be clear.

**Reply by qiangjiadong** [reply]
Thks Robin, finally I got good WIFI for exhibition, my hubs room works perfect! Check this: http://dungeon.hyperbody.space :))))
